Branch-SDK-TestBed/build.gradle

```diff
@@ -11,7 +12,7 @@ android {
     buildToolsVersion project.ANDROID_BUILD_TOOLS_VERSION
     
     defaultConfig {
-        minSdkVersion 7
+        minSdkVersion 8
```

Comment:
Is it possible to only change this for the app, and not for the SDK (it could be, but I'm not sure from here).